ui/ceval/src/view.ts

```diff
@@ -396,9 +410,10 @@ function renderPvWrapToggle(): VNode {
         const el = vnode.elm as HTMLElement;
         for (const event of ['touchstart', 'mousedown']) {
           el.addEventListener(event, (e: Event) => {
-            e.stopPropagation();
-            e.preventDefault();
-            $(el).closest('.pv').toggleClass('pv--nowrap');
+            e;
+            // e.stopPropagation();
+            // e.preventDefault();
+            // $(el).closest('.pv').toggleClass('pv--nowrap');
```

Comment:
What's going on here? I don't think this change is right?

Reply:
Yes it is not right, I was trying to learn the flow of code therefore I commented the code but forgot to uncomment the code.